### DISCUSSION TITLE
When the next release ??

### DISCUSSION THREAD

**Discussion by ZeyadMoustafaKamal** [Smithay/smithay#1490] [General]
So I looked at the last release and I saw that it was in 2021. I am wondering why there is no stable releases after this one even if the project is still being maintained very well. So just wondering about the future stable releases.

**Comment by Drakulix** [comment]
Because development currently is very fast and frequently breaks the api.

Releases will happen, when development speed slows down or a more stable api emerges.

(Or people step up to take care of this.)
